crates/types/src/view_sync.rs

```diff
@@ -0,0 +1,14 @@
+//! Types for the View Sync operations
+
+#[derive(PartialEq, PartialOrd, Clone, Debug, Eq, Hash)]
+/// Phases of view sync
+pub enum ViewSyncPhase {
```

Comment:
A similar question here. Why move this enum?

Reply:
This is needed in `types` due to the use in HotShotEvent